.github/workflows/continuous-integration.yml

```
@@ -9,11 +12,48 @@ jobs:
     runs-on: windows-latest
 
     steps:
-    - name: Install opensim-core
+    - uses: actions/checkout@v1
```

Might as well update this and other occurrences of `actions/checkout@v1`?
```suggestion
    - uses: actions/checkout@v2
```